org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -1907,6 +1911,23 @@ SsoIpamViewerPlus:
     principalId: !Ref ipamViewerPlusGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-viewer-plus-permission-set-arn' ]
 
+SsoMonitorCentralViewerPlus:
+  Type: update-stacks
+  DependsOn: SsoViewerPlus
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/SSO/aws-sso.yaml
```

Comment:
Is this not outdated? I see a .njk now.

Reply:
good catch.  updated to njk file. thanks.